### PR TITLE
Hotplug bridge interface with multiqueue

### PR DESCRIPTION
**Pull request by yalzhang**
Start VM with bridge type interface with multiqueue or hotplug bridge
tyep interface with multiqueue, then check on guest.

Signed-off-by: yalzhang <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
2 new case added:
virtual_network.iface_bridge.multiqueues.start.shared_physical_network
virtual_network.iface_bridge.multiqueues.hotplug_device.shared_physical_network

run all the cases locally passed
$ avocado run --vt-type libvirt  virtual_network.iface_bridge
/root/code/avocado/avocado/plugins/run.py:294: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : 0aa64941d02700192eb6d89971ad2faf71990979
JOB LOG    : /root/avocado/job-results/job-2020-08-03T01.57-0aa6494/job.log
 (1/5) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.default.shared_physical_network: PASS (127.89 s)
 (2/5) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.hotplug_iface.shared_physical_network: PASS (70.00 s)
 (3/5) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.hotplug_device.shared_physical_network: PASS (70.66 s)
 (4/5) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.multiqueues.start.shared_physical_network: PASS (58.66 s)
 (5/5) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.multiqueues.hotplug_device.shared_physical_network: PASS (73.54 s)
RESULTS    : PASS 5 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 402.60 s

**Review by chloerh — COMMENTED** [review]
Conflict?

**Comment by yalzhang** [comment]
> Conflict?

fixed, Thank you for reminding me!

File: libvirt/tests/src/virtual_network/iface_bridge.py [hunk 1]
```diff
@@ -241,30 +277,31 @@ def check_net_functions(guest_ip, ping_count, ping_timeout, guest_session, host_
...
 
         else:
-            # Start vm2 connect to the same bridge
-            mac2 = utils_net.generate_mac_address_simple()
-            vm2_iface_params = {"type": "bridge", "source": vm_iface_source, "filter": filter_name, "mac": mac2}
-            libvirt.modify_vm_iface(vm2_name, "update_iface", vm2_iface_params)
-            if vm2.is_alive():
-                vm2.destroy()
-            vm2.start()
+            if start_vm2:
```
Comment (chloerh) [inline libvirt/tests/src/virtual_network/iface_bridge.py:280]: I'm wondering, isn't 'start_vm2' True by default according to old code? After this change, 'start_vm2' is now False by default. Doesn't that mean 2 cases not need to start vm anymore? It's a change of test steps?

Comment (yalzhang) [reply libvirt/tests/src/virtual_network/iface_bridge.py:280]: Yes, the original case need to start vm2, others do not.

Comment (chloerh) [reply libvirt/tests/src/virtual_network/iface_bridge.py:280]: I mean 'hotplug_iface' and 'hotplug_device' these 2 cases, do they need to 'start_vm2' too?

**Review by chloerh — COMMENTED** [review]
See comments

File: libvirt/tests/src/virtual_network/iface_bridge.py (hunk 1 above)
Comment (yalzhang) [reply libvirt/tests/src/virtual_network/iface_bridge.py:280]: no

Comment (chloerh) [reply libvirt/tests/src/virtual_network/iface_bridge.py:280]: Ok then

**Review by chloerh — APPROVED** [review]
I guess it needs another approval to be merged.

**Review by chunfuwen — APPROVED** [review]

**Pull request merged**